packages/block-editor/src/hooks/style.js

```diff
@@ -157,23 +157,34 @@ const skipSerializationPaths = {
 	],
 };
 
+const skipSerializationPathsSave = {
+	...skipSerializationPathsEdit,
+	[ `${ SPACING_SUPPORT_KEY }` ]: [ 'spacing', 'blockGap' ],
```

Comment:
Note for myself to fix up tomorrow: this currently breaks serialization of the padding support (e.g. test Padding on the Group block)

Reply:
Resolved in: https://github.com/WordPress/gutenberg/pull/33991/commits/3eac45c18c734824fc2bcdf93ac5d42ab51013ba